include/grisp.hrl

```diff
@@ -0,0 +1,17 @@
+-ifndef(GRISP_HRL).
```

Comment:
Does the `-ifndef` mean that if `GRISP_HRL` is already defined, we silently skip all this logic here?
Wouldn't it be better to let the defines clash to hopefully get a compiler error?

Reply:
@matlaj This is a common pattern to allow multiple includes from different places and have the code only included once